### PR TITLE
fix: broken links and clarify check-in

### PR DESCRIPTION
**Pull request by wass3r**
fixing a few broken links.

the TODO didn't have a valid target page, so I took some liberties to rework it. i cross-referenced the check-durations on the server and worker since there should be some coordination there so you get somewhat accurate values. 

while doing so, i realized the defaults might need updating in code to match better? with the current defaults, it's quite possible your active worker count is off.

### CODE REVIEW

File: content/installation/server/reference/_index.md [hunk 1]
```diff
@@ -598,10 +598,13 @@ This variable has a default value of `/account/authenticate`.
...
+This variable sets the interval of time the workers will be considered active. A worker is considered active if it has registered with the server inside the give duration.
 
 The variable can be provided as a `duration` (i.e. `5s`, `10m`).
 
 {{% alert title="Note:" color="primary" %}}
-This variable has a default value of `5m`.
+This variable has a default value of `5m`.\
+\
+The value should coordinate with the [`VELA_CHECK_IN`](/docs/installation/worker/reference/#vela_check_in)
+setting provided to the [worker](/docs/installation/worker/)
```
Comment (jbrockopp) [inline content/installation/server/reference/_index.md]: I think we want to condense this into one line and missing `.`:

```suggestion
The value should coordinate with the [`VELA_CHECK_IN`](/docs/installation/worker/reference/#vela_check_in) setting provided to the [worker](/docs/installation/worker/).
```

File: content/installation/worker/reference/_index.md [hunk 2]
```diff
@@ -110,7 +110,10 @@ This variable sets the maximum duration of time a worker will wait before regist
 The variable can be provided as a `duration` (i.e. `5s`, `10m`).
 
 {{% alert title="Note:" color="primary" %}}
-This variable has a default value of `15m`.
+This variable has a default value of `15m`.\
+\
+The value should coordinate with the [`VELA_WORKER_ACTIVE_INTERVAL`](/docs/installation/server/reference/#vela_worker_active_interval)
+setting provided to the [server](/docs/installation/server/)
```
Comment (jbrockopp) [inline content/installation/worker/reference/_index.md]: one line and missing `.`:

```suggestion
The value should coordinate with the [`VELA_WORKER_ACTIVE_INTERVAL`](/docs/installation/server/reference/#vela_worker_active_interval) setting provided to the [server](/docs/installation/server/).
```

**Review by jbrockopp — CHANGES_REQUESTED** [review]

**Comment by wass3r** [comment]
changes made - thanks @jbrockopp

**Review by jbrockopp — APPROVED** [review]
LGTM

**Review by plyr4 — APPROVED** [review]

**Pull request merged**